problems/arraysIntro.js

```diff
@@ -64,29 +85,59 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
+const largerArray = (array1, array2) => {
+  if (array1.length > array2.length) {
+    return array1;
+  } else if (array1.length < array2.length) {
+    return array2;
+  } else {
+    return "They are the same size";
+  }
+};
+
+// console.log(largerArray([1, 4, 5, 6], [1, 2, 3, 4, 5, 6]));
+// console.log(largerArray([1, 2, 3, 4, 5, 6], [1, 2, 6]));
+// console.log(largerArray([1, 2, 3, 4, 5, 6], [1, 2, 3, 4, 5, 6]));
 
 /**
  * Takes in an array and returns the middle element.
  * If the array is even in length return the middle two elements in their own array.
  * @param {Array} array
  * @returns {*|Array}
  */
-function middleElement() {}
+
+// const middleElement = (array) => {
+//   if (array.length % 2 !== 0) {
+//     return array[(array.length - 1) / 2];
+//   } else {
+//     return [array[array.length / 2 - 1], array[array.length / 2]];
+//   }
+// };
+
+const middleElement = (array) => { // ES6+ ternary
+  return array.length % 2 === 0
+    ? [array[array.length / 2 - 1], array[array.length / 2]]
```

Comment:
love that you're using the ternary but it's making this a little hard to read. For this I'd save array.length-1/2 and arrray.length-1/2 -1 in variables to make for a cleaner return statement.